website/docs/r/quicksight_group.html.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: "aws"
+page_title: "AWS: aws_quicksight_group"
+sidebar_current: "docs-aws-resource-quicksight-group"
+description: |-
+  Manages a Resource Quick Sight Group.
+---
+
+# aws_quicksight_group
```

Comment:
We likely added this to the other resource pages after this PR was submitted

```suggestion
# Resource: aws_quicksight_group
```

Reply:
fixed